conf/application.conf

```diff
@@ -186,6 +193,19 @@ microservice {
       port = 9508
       enabled = true
     }
+
+    secure-data-exchange-proxy {
+      host = localhost
+      port = 8764
+      file-ready-uri = "/notification/fileready"
+    }
+
+    sdes-stub {
+       host = localhost
+       port = 9191
+       file-ready-uri = "/sdes-stub/notification/fileready"
+    }
```

Comment:
This shouldn't exist. Stub details should replace those of the proxy.

I imagine what you've done is copy the message parser. You don't need to do that here, because I did that to ensure that we could use the proxy properly. I wouldn't worry about that here.